runtime/bindings/python/iree/runtime/array_interop.py

```diff
@@ -114,7 +117,12 @@ def _transfer_to_host(self, implicit):
                 "DeviceArray cannot be implicitly transferred to the host: "
                 "if necessary, do an explicit transfer via .to_host()"
             )
-        self._mapped_memory, self._host_array = self._map_to_host()
+        bv_property = repr(self._buffer_view)
```

Comment:
we don't want to be stringifying the buffer view for this - you can ask the buffer for its memory type

Reply:
Done!